bundles/com.espressif.idf.launch.serial.core/src/com/espressif/idf/launch/serial/util/ESPFlashUtil.java

```diff
@@ -119,7 +119,7 @@ public static String getEspJtagFlashCommand(ILaunchConfiguration configuration)
 			{
 				buildPath = buildPath.substring(1);
 			}
-			espFlashCommand = espFlashCommand.replace("<path-to-build-dir>", buildPath); //$NON-NLS-1$
+			espFlashCommand = espFlashCommand.replace("<path-to-build-dir>", buildPath.replace(" ", "\\ ")); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
```

Comment:
Can you check if we can use double quotes around the path to avoid these issues?

`espFlashCommand = espFlashCommand.replace("<path-to-build-dir>", "\"" + buildPath + "\"");
`